presidio-image-redactor/presidio_image_redactor/dicom_image_pii_verify_engine.py

```diff
@@ -45,12 +45,14 @@ def verify_dicom_instance(
         self,
         instance: pydicom.dataset.FileDataset,
         padding_width: int = 25,
+        display_image: bool = True,
         **kwargs,
     ) -> Tuple[PIL.Image.Image, dict, list]:
```

Comment:
you can use the `Optional` type hint to make it clear to the client that there is a possibility for an image to not be returned

Reply:
@niwilso I mean the return type :)
Tuple[Optional[PIL.Image.Image], dict, list]